src/Umbraco.Web.BackOffice/Controllers/DashboardController.cs

```diff
@@ -116,8 +131,16 @@ public async Task<JObject> GetRemoteDashboardContent(string section, string base
             return result;
         }
 
+        // TODO(V10) : consider removing baseUrl as parameter
         public async Task<IActionResult> GetRemoteDashboardCss(string section, string baseUrl = "https://dashboard.umbraco.org/")
         {
+            if (!IsAllowedUrl(baseUrl))
+            {
+                var errorMsg = $"The following URL is not listed in the setting 'Umbraco:CMS:ContentDashboard:ContentDashboardUrlAllowlist' in configuration: {baseUrl}";
+                _logger.LogError(errorMsg);
+                return BadRequest(errorMsg);
```

Comment:
I don't think we should include the full error message in the error response, only in the log entry, and instead just return something more generic like "Could not load dashboard"